### PR TITLE
Adding funcionality to show the number of available parking

### PR DESCRIPTION
**Pull request by Lubandeo**
## Description

Add functionality to persist the number of parking available using the button created in step i.

## Story Type

- New feature  :white_check_mark: 
- Change request
- Bug fix
- Tech

## How Can This Be Tested?

By clickling on the button called "Number of parking available" and checking if the correct number is shown. 

## Evidences

|   |Nexus 5  | Pixel 4 |
| ------------- | ------------- | ------------- |
| before pressing the button  | image|image
| after pressing the button |image | image |
## Checklist

- All unit Test passed  :white_check_mark:

## Tested on

Tested on Nexus 5 and Pixel 4

### CODE REVIEW

File: app/src/main/java/com/mobile/tandil/javabaseproject/mvp/view/base/ActivityView.java [hunk 1]
```diff
@@ -1,19 +1,28 @@
 package com.mobile.tandil.javabaseproject.mvp.view.base;
 
 import android.app.Activity;
+import android.content.Context;
+
```
Comment (jnicolaslaurenti) [inline app/src/main/java/com/mobile/tandil/javabaseproject/mvp/view/base/ActivityView.java]: remove unnecessary new lines

Comment (Lubandeo) [reply app/src/main/java/com/mobile/tandil/javabaseproject/mvp/view/base/ActivityView.java]: Done

**Review by jnicolaslaurenti — APPROVED** [review]

File: app/src/main/java/com/mobile/tandil/javabaseproject/mvp/model/ParkingModel.java [hunk 2]
```diff
@@ -3,4 +3,16 @@
 import com.mobile.tandil.javabaseproject.mvp.contract.ParkingContract;
 
 public class ParkingModel implements ParkingContract.Model{
+
+    private int parkingAvailable = 0;
```
Comment (Georgina-Fernandez) [inline app/src/main/java/com/mobile/tandil/javabaseproject/mvp/model/ParkingModel.java]: Use an invalid value as default, for example -1

File: app/src/main/java/com/mobile/tandil/javabaseproject/mvp/view/ParkingView.java [hunk 3]
```diff
@@ -1,12 +1,21 @@
...
-import android.app.Activity;
 
 public class ParkingView extends ActivityView implements ParkingContract.View {
 
-    public ParkingView(Activity activity) {
+    private final ActivityMainBinding binding;
+
+    public ParkingView(Activity activity, ActivityMainBinding binding) {
         super(activity);
+        this.binding=binding;
```
Comment (mariano98montero) [inline app/src/main/java/com/mobile/tandil/javabaseproject/mvp/view/ParkingView.java]: Please remember to format your code

File: app/src/main/java/com/mobile/tandil/javabaseproject/mvp/model/ParkingModel.java (hunk 2 above)
Comment (Lubandeo) [reply app/src/main/java/com/mobile/tandil/javabaseproject/mvp/model/ParkingModel.java]: Done

File: app/src/main/java/com/mobile/tandil/javabaseproject/mvp/view/ParkingView.java (hunk 3 above)
Comment (Lubandeo) [reply app/src/main/java/com/mobile/tandil/javabaseproject/mvp/view/ParkingView.java]: Done

File: app/src/main/java/com/mobile/tandil/javabaseproject/mvp/view/base/ActivityView.java [hunk 4]
```diff
@@ -9,11 +10,17 @@ public class ActivityView {
     private WeakReference<Activity> activityRef;
 
     public ActivityView(Activity activity) {
+
```
Comment (DiegoDiazGlobant) [inline app/src/main/java/com/mobile/tandil/javabaseproject/mvp/view/base/ActivityView.java]: Remove extra line please

Comment (Lubandeo) [reply app/src/main/java/com/mobile/tandil/javabaseproject/mvp/view/base/ActivityView.java]: Done

**Review by DiegoDiazGlobant — APPROVED** [review]

**Review by Georgina-Fernandez — APPROVED** [review]

**Review by mariano98montero — APPROVED** [review]

**Review by jnicolaslaurenti — APPROVED** [review]

**Pull request merged**